src/amo/components/AddonMoreInfo.js

```diff
@@ -0,0 +1,75 @@
+import React, { PropTypes } from 'react';
+import { compose } from 'redux';
+
+import Link from 'amo/components/Link';
+import translate from 'core/i18n/translate';
+
+import './AddonMoreInfo.scss';
+
+
+export class AddonMoreInfoBase extends React.Component {
+  static propTypes = {
+    addon: PropTypes.object.isRequired,
+    i18n: PropTypes.object.isRequired,
+  }
+
+  render() {
+    const { addon, i18n } = this.props;
+
+    if (!addon) {
+      return null;
+    }
+
+    // TODO: Use the addonType for the privacy text, so it reads
+    // "for this extension", "for this theme", etc.
+    return (
+      <section className="MoreInfo">
+        <h2 className="MoreInfo-header">{i18n.gettext('More information')}</h2>
+
+        <dl className="MoreInfo-contents">
+          {addon.homepage ? <dt>{i18n.gettext('Website')}</dt> : null}
+          {addon.homepage ? (
+            <dd>
+              <a href={addon.homepage}
+                ref={(ref) => { this.homepageLink = ref; }}>{addon.homepage}</a>
+            </dd>
+          ) : null}
+          <dt>{i18n.gettext('Version')}</dt>
+          <dd ref={(ref) => { this.version = ref; }}>
+            {addon.current_version.version}
+          </dd>
+          <dt>{i18n.gettext('Last updated')}</dt>
+          <dd>{addon.last_updated}</dd>
```

Comment:
yes, filing small issues to fix up things like this is the best way to keep moving forward IMO